geoips/plugins/modules/output_checkers/netcdf.py

```diff
@@ -204,34 +203,32 @@ def outputs_match(plugin, output_product, compare_product):
             mindiff = (compare_xobj[varname] - out_xobj[varname]).min()
             meandiff = (compare_xobj[varname] - out_xobj[varname]).mean()
             if mindiff != 0:
-                LOG.interactive(f"    *** mindiff {varname}: {mindiff} ***")
+                log_with_emphasis(LOG.interactive, "mindiff {varname}: {mindiff}")
                 diffout += [f"mindiff {varname}: {mindiff}\n"]
             if maxdiff != 0:
-                LOG.interactive(f"    *** maxdiff {varname}: {maxdiff} ***")
+                log_with_emphasis(LOG.interactive, "mindiff {varname}: {maxdiff}")
                 diffout += [f"maxdiff {varname}: {maxdiff}\n"]
             if meandiff != 0:
-                LOG.interactive(f"    *** meandiff {varname}: {meandiff} ***")
+                log_with_emphasis(LOG.interactive, "mindiff {varname}: {meandiff}")
```

Comment:
Maybe it would be better for these to be done in a single call to log_with_emphasis to avoid too many sets of asterisk boxes. Alternatively, maybe log_with_emphasis should have an option to turn off the leading and trailing lines.

Reply:
Fixed!